src/BuiltInTools/BrowserRefresh/Microsoft.AspNetCore.Watch.BrowserRefresh.csproj

```diff
@@ -14,4 +14,11 @@
     <EmbeddedResource Include="BlazorHotReload.js" />
   </ItemGroup>
 
+  <!-- Reference 6.0.2 Asp.Net targeting pack in Source Build - this was the earliest version after the API surface stabilized -->
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
+    <FrameworkReference
```

Comment:
Does this affect the Microsoft.NETCore.App.Ref version as well?  <Usage Id="Microsoft.NETCore.App.Ref" Version="6.0.20" /> is being reported as a prebuilt.  Wondering if a another FwRef is needed to address that?

Reply:
It shouldn't - what version is being reported as a prebuilt? We can hard-code to 6.0.0 if we need to

Reply:
```
    <Usage Id="Microsoft.NETCore.App.Ref" Version="6.0.20" />
```

Reply:
This should be 6.0.2 to match AspNetCore. That's the version I added in SBRP.

Reply:
Updated